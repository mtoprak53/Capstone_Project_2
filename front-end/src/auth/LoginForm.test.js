import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import LoginForm from "./LoginForm";

it("matches snapshot", function () {
  const { asFragment } = render(
      <MemoryRouter>
        <LoginForm />
      </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
