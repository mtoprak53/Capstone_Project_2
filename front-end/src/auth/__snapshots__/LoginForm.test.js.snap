// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="LoginForm"
  >
    <div
      class="container col-md-6 offset-md-3 col-lg-4 offset-lg-4"
    >
      <h3
        class="mb-3"
      >
        Log In
      </h3>
      <div
        class="card"
      >
        <div
          class="card-body"
        >
          <form>
            <div
              class="form-group"
            >
              <label>
                Username
              </label>
              <input
                autocomplete="username"
                class="form-control"
                name="username"
                required=""
                value=""
              />
            </div>
            <div
              class="form-group"
            >
              <label>
                Password
              </label>
              <input
                autocomplete="current-password"
                class="form-control"
                name="password"
                required=""
                type="password"
                value=""
              />
            </div>
            <button
              class="btn btn-primary float-right"
            >
              Submit
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
